packages/opentelemetry-types/src/trace/tracer.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Span } from './span';
+import { Attributes } from './attributes';
+import { SpanKind } from './span_kind';
+import { SpanContext } from './span_context';
+
+/**
+ * Options needed for span creation
+ *
+ * @todo: Move into module of its own
+ */
+export interface SpanOptions {
+  kind?: SpanKind;
```

Comment:
It would be nice to see block comment (`/** … */`) for each of these properties similar to other interfaces.